content/kit/pi/update-file.md

```diff
@@ -0,0 +1,27 @@
+---
+title: The Update File
+---
+
+The update file is the simplest way to update your Pi with new software from us.  
+
+{{% notice tip %}}
+Using this file is the recommended way to update your kit.
+{{% /notice %}}
+
+## Using the update file
+
+1. Put the `update.tar.xz` file on a USB drive
```

Comment:
We should clarify that they need to have at least the power-board plugged in. I'd actually like to see us remove this requirement as it complicates the process of updating pis in development, but it's a requirement for now.

Reply:
Agreed, that's an annoying _feature_. I'll add that a minimal robot setup is required